src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -4676,6 +4681,10 @@ subroutine gsi_fv3ncdf_write(grd_ionouv,cstate_nouv,add_saved,filenamein,fv3file
                 call fv3_ll_to_h(work_a(:,:),work_b(:,:),nloncase,nlatcase,nlon_regional,nlat_regional,grid_reverse_flag)
              endif
           endif
+          if (ifindstrloc(vnames_nonnegativetracers,trim(varname))> 0) then
+              write(6,*)'thinkdeb varname for nonegative is ',trim(varname)
```

Comment:
@AmandaBack-NOAA  Could you remove this writing out line which was for debugging  ?  Or if you want to keep this line, please polish it and only output on one MPI process.

Reply:
@TingLei-NOAA Removed.